rxmongo/src/main/scala/akka/contrib/persistence/mongodb/RxMongoReadJournaller.scala

```diff
@@ -123,7 +127,8 @@ class CurrentAllEvents(val driver: RxMongoDriver) extends IterateeActorPublisher
     journal.find(BSONDocument())
       .projection(BSONDocument(EVENTS -> 1))
       .cursor[BSONDocument]()
-      .enumerate()
+      .enumerator()
+      .map(doc => doc) // this is needed for suffix collections
```

Comment:
I do get those exact errors.  It looks like a bug in that new rxmongo `iteratee` library.  Or at a minimum a regression in behavior from `.enumerate()`, which works without the `map(identity)` despite the deprecation warning.

Let's go back to the warnings by using the deprecated `enumerate` and remove the not-quite-ready rxmongo extension library from the build.